app/models/assignment.rb

```diff
@@ -677,6 +677,7 @@ def summary_json(user)
                     .where('memberships.user_id': user.id)
     end
 
+    maximum_mark = max_mark(user.admin? ? :all : :ta)
```

Comment:
You want `:ta` here, which is the default (the `:all` and `:peer` are only used for peer reviews).